### PR TITLE
no species found

### PR DESCRIPTION
**Pull request by utas-tajona**

### CODE REVIEW

**Comment by utas-tajona** [comment]
https://github.com/aodn/backlog/issues/2933

File: api/src/main/java/au/org/aodn/nrmn/restapi/validation/process/RawValidation.java [hunk 1]
```diff
@@ -91,6 +91,7 @@
                         Tuple2.of("Latitude", new DoubleFormatValidation(StagedRow::getLatitude, "Latitude")),
                         Tuple2.of("Longitude", new DoubleFormatValidation(StagedRow::getLongitude, "Longitude")),
                         Tuple2.of("Direction", new DirectionDataCheck()),
+                        Tuple2.of("found",new SpeciesNotFoundCheck()),
```
Comment (jonescc) [inline api/src/main/java/au/org/aodn/nrmn/restapi/validation/process/RawValidation.java:94]: Need an import for this - getting a build error.

**Review by jonescc — CHANGES_REQUESTED** [review]
Looks good to me - but we need to fix that build error.

**Review by jonescc — APPROVED** [review]
Works OK for non-zero measures for species not found but doesn't check total/inverts as per spec.

Merging as the measure checks work and noting this on backlog item for @bpasquer to clarify.

**Comment by jonescc** [comment]
Also I note that we get more than one species heading now - one for this check and one for Species not found checks.  Not sure if that's important.

**Comment by jonescc** [comment]
image

**Pull request merged**